lib/Dialect/TritonGPU/Transforms/AccelerateAMDMatmul.cpp

```diff
@@ -0,0 +1,251 @@
+#include "mlir/IR/TypeUtilities.h"
```

Comment:
@zhanglx13

I've Separated this code from common AccelerateMatmul pass, so I can add an additional option to it.

Do you think it is ok to do this in this PR or is it better to separate it?